packages/jest-emotion/README.md

```diff
@@ -16,7 +16,7 @@ The easiest way to test React components with emotion is with the snapshot seria
 // jest.config.js
 module.exports = {
   // ... other config
-  snapshotSerializers: ['jest-emotion']
+  snapshotSerializers: ['jest-emotion', /* ... other snapshotSerializers */ ]
```

Comment:
```suggestion
  snapshotSerializers: ['jest-emotion', /* if needed other snapshotSerializers should go here */ ]
```

Reply:
I've wanted to commit this suggestion through the GitHub interface - but it seems like I don't have sufficient permissions to push to your fork.

You can check how to allow this for the future PRs here - https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork . At the moment just commit this suggestion with a click on a button.